### PR TITLE
Updates and fixes

### PR DESCRIPTION
**Pull request by Maze-Mind**
* FIX: Re-added clickhouse link
* FIX: Fixed sampling page routing
* FIX: Reverted changes to config/config.js -- It broke lookup api
* FIX: Removed FastGraph from lookup page -- It's still too slow for it
* NEW: Added a page for FastGraph under /fastgraph path
* NEW: Added range bar for clickhouse limit
* NEW: Added map buttons to search and map results

### CODE REVIEW

**Comment by audrism** [comment]
Awesome!

**Comment by audrism** [comment]
- @Maze-Mind @branson2015 I am wondering if the code you have in the PR has the latest fastest fastgraph implementation

- Also, fastgraph route is not visible from the toolbar

- finally, if i go to it by typing into url, it seems to have issues if I paste a commit sha1 there

**Comment by Maze-Mind** [comment]
I used the latest implementation of fastgraph that was in master at the time. I didn't make a link to it because it wasn't working yet -- It is trying to do mappings using version R which I think is why it isn't working. Once we get it working I can make a link to it in the toolbar.

**Comment by branson2015** [comment]
I had to add a bash script to swsc/lookup repo that was not included in the
DRE PR. It was basically a one-liner wrapper around the perl script - I
could probably remove this, but the existing code used this same
architecture, so I just followed suit. This is probably why it does not
work if you go to it directly by the url, but, rest assured, I do have a
version of fastgraph working with the latest perl script additions.

**Comment by audrism** [comment]
Great I merged your the PR.

Is there a way to send objects as standard input, however. 
For example, getNeighborhood author name might be very hard to parse from the command line?

So it would be easier if it was provided as standard input

**Comment by audrism** [comment]
Ok, I added author, project, file, tree, blob search as the parameter types a, p, f, tree, blob
